web/app/js/components/ResourceDetail.jsx

```diff
@@ -84,7 +84,8 @@ export class ResourceDetailBase extends React.Component {
       resourceIsMeshed: true,
       pendingRequests: false,
       loaded: false,
-      error: null
+      error: null,
+      queryForDefinition: false,
```

Comment:
could you add a note above this line that explains that we are setting the variable to false now because we are not displaying the resource definition, but that this can change in the future?

Reply:
Done